test/test_events.py

```diff
@@ -350,9 +336,7 @@ def test_service_pooled_events(rabbit_manager, rabbit_config,
                            properties=dict(content_type='application/json'))
 
     # a total of two events should be received
-    with eventlet.timeout.Timeout(EVENTS_TIMEOUT):
```

Comment:
They assumed that the events would not be received immediately and we'd need to wait until they were processed.

As it happens, `rabbit_manager.publish` appears to always deliver the message before returning, so they're not required. I'll add resilience if this isn't reliably the case, but I'd rather not unless I have to.